ngi_visualizations/preseq_complexity_curves/plot_complexity_curves.py

```diff
@@ -25,10 +25,19 @@ def plot_complexity_curves(ccurves, real_counts_path=False, output_name='complex
     the second case "preseq lc_extrap" should be usued. Please, refer to preseq manual available at
     http://smithlabresearch.org/wp-content/uploads/manual.pdf for examples (pages 12 to 14 are the most informatives ones)
     """
-    
+
     if x_min < 0 or x_max <= x_min:
         sys.exit("problem with x-min or x-max ({}, {}). x-min must be equal or higher to 0 and less than x-max".format(x_min, x_max))
-    
+
+    # Covert limit counts to coverage
+    if coverage > 0:
+        if read_length == 0:
```

Comment:
You should be able to integrate this logic whit ArgumentParser, not  a big thing but makes code cleaner... sorry for not coming back to you @ewels :-D